modules/class-simplechart-template.php

```diff
@@ -149,4 +150,18 @@ public function filter_insert_chart( $content ) {
 
 		return $template_html . $content;
 	}
+
+	public function iframe_scripts() {
+		do_action( 'wp_print_footer_scripts' );
+	}
+
+	public function default_placeholder( $id, $content ) {
+		if ( ! $content ) {
+			$src = esc_url( Simplechart::instance()->get_plugin_url( '/images/placeholder.png' ) );
+			return '<amp-img layout="fixed" width="311" height="61" src="'
+				. $src . '" placeholder></amp-img>';
```

Comment:
We generally prefer to escape as late as possible, i.e. 
```
$src = Simplechart::instance()->get_plugin_url( '/images/placeholder.png' );
return '<amp-img layout="fixed" width="311" height="61" src="'
    . esc_url( $src ) . '" placeholder></amp-img>';
```
although in this case you could just do
```
return sprintf(
    '<amp-img layout="fixed" width="311" height="61" src="%s" placeholder></amp-img>',
    esc_url( Simplechart::instance()->get_plugin_url( '/images/placeholder.png' ) )
);
```
Also, can it be a self-closing tag `<amp-img />`?